app/src/main/res/css/modules/compact_user_card.sass

```diff
@@ -51,8 +52,8 @@
       cursor: pointer
   .logout
     position: absolute
-    bottom: 5px
-    left: 10px
+    bottom: 10px
+    left: 1.5rem
```

Comment:
Are you referring to the `left` property, or the `bottom` property?
Could you comment the suitable values for both of them?